addons/wp/CfgWaypoints.hpp

```diff
@@ -11,42 +11,42 @@ class CfgWaypoints {
         class EGVAR(danger,Garrison) {
             displayName = "1 TASK GARRISON";
             displayNameDebug = QEGVAR(danger,Garrison);
-            file = QPATHTOF(functions\fnc_taskGarrison.sqf);
+            file = QPATHTOF(scripts\fnc_wpGarrison.sqf);
```

Comment:
Missing file

Reply:
A brief description of the fnc_wpXxx.sqf  files

These adapt information from the waypoint: group, position and radius,  as well as any future WP specific settings and variables-- on the model of the 'advanced mine sweeping' wp in vanilla,  and parse them into the relevant script. 

1. This divide means that the taskXxx function can remain unencumbered by any WP specific checks. It is a function which can be called anywhere without any connection to a specific waypoint.  As noted above. This adds utility and makes maintaining the function in a 'universal' manner much easier. 

2. Noted exception is the 'Register Artillery' wp-- which really only does one thing.  It might be moved into script, but it is conceivably something that can be called as a function by MM.  

3. There is nothing wrong with moving all /scripts into /functions and compiling them. I just did not see the need for three reasons: 
3.1 - Intended use outlined above. 
3.2- Easier to keep distinct in the design process. 
3.3- Not necessary to keep constantly in memory. 

-k

Reply:
fixed